src/dev/bazel/jsts_transpiler.bzl

```diff
@@ -0,0 +1,36 @@
+"Simple wrapper over @babel/cli so we can quickly re-use the same configurations over packages"
+
+load("@npm//@babel/cli:index.bzl", _babel = "babel")
+
+def jsts_transpiler(name, srcs, build_pkg_name, root_input_dir = "src", config_file = ".babelrc", additional_args = ["--quiet"], **kwargs):
```

Comment:
Curious why `jsts`. Seems `tsjs` might make more sense as it's transpiling TS to JS.

Reply:
the idea is that this is a js and a ts transpiler and not a js to ts transpiler. Do you think `js_ts_transpiler` would make more sense? Or even just `code_transpiler`?

Reply:
Gotcha, in that case, I think this is fine.